woocommerce/woocommerce-android
# TTP: Adds handling of ttp universal link to the app

Closes: #8459


### Description


Adds handling of the TTP universal link to the app

> https://woocommerce.com/mobile/payments/tap-to-pay

### Testing instructions

* Go to the app's info in the system ->  Set as default -> Supported web addresses -> check on woocommerce.com (@wzieba That's weird that this has to be done even on release build 🤔. The build that downloaded from google play doesn't have check there, so woocommerce.com is supported web address by default. Maybe you know anything about that?)
* To trigger the event ADB can be used:

```
 adb shell am start -a android.intent.action.VIEW \
   -c android.intent.category.BROWSABLE \
   -d "https://woocommerce.com/mobile/payments/tap-to-pay"
```

or alternatively, copy the link into an app that supports links, e.g. send the link to your email/chat
* Click on the link
* Notice that the TTP summary is opened with correct back stack (the same as with normal app usage)

---


Repeat with:
```
https://woocommerce.com/mobile/payments/tap-to-pay?blog_id={current blog id}
https://woocommerce.com/mobile/payments/tap-to-pay?blog_id={blog id owned by you but not selected}
https://woocommerce.com/mobile/payments/tap-to-pay?blog_id={incorect blog id}
```
In the first case: the current blog id should be selected
In the second case: provided blog id should be selected
In the third case: My Store should be shown

### Images/gif



https://user-images.githubusercontent.com/4923871/222707041-0ee84b8f-5100-4e79-b9cc-592814438a71.mp4
